red.py

```diff
@@ -174,6 +176,32 @@ def user_allowed(self, message):
         else:
             return True
 
+    async def pip_install(self, name, *, timeout=60):
```

Comment:
Having a timeout here is unwise, something like PyNaCl will take at least that long to build.